Source/control.cpp

```diff
@@ -1612,31 +1625,31 @@ void control_print_info_str(int y, char *str, BOOL center, int lines)
 {
 	BYTE c;
 	char *tmp;
-	int screen_x, line, nOffset;
+	int strWidth, lineOffset, lineStart;
 
-	line = 0;
-	nOffset = lineoffset[y + 4 * lines + lines] + (SCREEN_WIDTH - PANEL_WIDTH) / 2;
+	ceterOfffset = 0;
+	lineStart = lineOffsets[lines][y] + (SCREEN_WIDTH - PANEL_WIDTH) / 2;

```

Comment:
It is bin exact also with this change?

Reply:
Yes, but I broke the build before going to bed :)